src/ServiceControl/Infrastructure/Nancy/Modules/RootModule.cs

```diff
@@ -27,21 +27,58 @@ public RootModule()
                     Name = SettingsReader<string>.Read("Name", "ServiceControl"),
                     Description = SettingsReader<string>.Read("Description", "The management backend for the Particular Service Platform"),
                     LicenseStatus = License.IsValid ? "valid" : "invalid",
-                    InstanceInfo = BaseUrl + "/instance-info"
+                    Configuration = BaseUrl + "/configuration"
                 };
 
                 return Negotiate
                     .WithModel(model);
             };
 
-            Get["/instance-info"] = p => Negotiate
-                    .WithModel(new
-                                  {
-                                      WindowsService = Settings.ServiceName,
-                                      LogfilePath = Path.Combine(Settings.LogPath, "logfile.txt"),
-                                      Settings.TransportType,
-                                      RavenDBPath = Settings.DbPath
-                                  });
+            Func<dynamic, dynamic> configuration = p => Negotiate
+                .WithModel(new
+                {
+                    Host = new
+                    {
+                        Settings.ServiceName,
+                        RavenDBPath = Settings.DbPath,
+                        Logging = new
+                        {
+                            LoggingSettings.LogPath,
```

Comment:
Should we prefer explicit property names here? i.e. `LogPath = LoggingSettings.LogPath`. Otherwise refactoring the settings class will impact the public json API and we won't detect that straight away. The same comment holds for the rest of this method.